README.md

```diff
@@ -153,6 +153,60 @@ acceptance_models = acceptance_model_creator.create_acceptance_map_per_observati
+                                                      offset_axis_acceptance,
+                                                      exclude_regions=exclude_regions,
+                                                      oversample_map=10,
+                                                      min_run_per_cos_zenith_bin=3,
+                                                      initial_cos_zenith_binning=0.01)
+acceptance_models = acceptance_model_creator.create_acceptance_map_per_observation(obs_collection,
+                                                                                   off_observations=obs_collection_off,
+                                                                                   zenith_binning=True,
+                                                                                   zenith_interpolation=True)
+```
+
+## Store background model for later application
+
+It could be in some case useful to precompute a model and applying it on data later. The example below cover the case where you want to use a model per run using zenith interpolation and OFF runs.
+
+However, it's possible to use this functionality without OFF runs or zenith interpolation. In the last case you just need to provide a model in a gammapy format.
+
+There are at this stage no define file format for storing the intermediate results, we suggest to store the dictionary created directly using pickle.
+
+### Creating and storing the model
+```python
+import pickle
+acceptance_model_creator = RadialAcceptanceMapCreator(energy_axis_acceptance,
+                                                      offset_axis_acceptance,
+                                                      exclude_regions=exclude_regions,
+                                                      oversample_map=10,
+                                                      min_run_per_cos_zenith_bin=3,
+                                                      initial_cos_zenith_binning=0.01)
+base_model = acceptance_model_creator.create_model_cos_zenith_binned(obs_collection_off)
+
+with open('my_bkg_model.pck', mode='wb') as f:
+    pickle.dump(base_model, f)
+```
+
+### Loading and applying the model
+```python
+import pickle
+with open('my_bkg_model.pck', mode='rb') as f:
+    base_model = pickle.load(f)
+    
+acceptance_model_creator = RadialAcceptanceMapCreator(energy_axis_acceptance,
+                                                      offset_axis_acceptance,
+                                                      exclude_regions=exclude_regions)
+acceptance_models = acceptance_model_creator.create_acceptance_map_per_observation(obs_collection,
+                                                                                   base_model=base_model,
+                                                                                   zenith_binning=True,
+                                                                                   zenith_interpolation=True)
```

Comment:
If you want to keep as is, I would still split in three section instead of two :
### Creating the model

### Storing and loading a model with pickle

### Applying a model in memory

Reply:
I've made some update to the documentation including this point. Let me know if you have further comments.